config/routes.rb

```diff
@@ -0,0 +1,18 @@
+Rails.application.routes.draw do
+  get '/welcome', to: 'welcome#index', as: 'welcome'
+  get 'entities/new'
+  get 'entities/destroy'
+  get 'entities/edit'
+  get 'entities/update'
+  get 'entities/index'
+  devise_for :users, sign_out_via: [:get, :post], controllers: { registrations: "users/registrations" }
+  resources :users, only: [:index, :show, :destroy] do
+    resources :groups do
+      resources :entities
+    end
```

Comment:
- [OPTIONAL] It would be nice to use only and except options to add only necessary routes in case maybe there are some routes that are not used. 👍🏻